src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-alert */
+import { Link, useNavigate } from 'react-router-dom';
+import { useState } from 'react/cjs/react.development';
+
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+import { inputText, buttonText, serverAddress } from '../../constants';
+
+import './Registration.css';
+
+async function performRegistration(userName, userEmail, userPassword) {
+  const registrationData = {
+    name: userName,
+    email: userEmail,
+    password: userPassword,
+  };
+
+  const response = await fetch(`${serverAddress}/register`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(registrationData),
+  });
+  const responseJSON = await response.json();
+  return responseJSON;
+}
+
+export default function Registration() {
+  const [userName, setUserName] = useState('');
+  const [userEmail, setUserEmail] = useState('');
+  const [userPassword, setUserPassword] = useState('');
+  const navigate = useNavigate();
+
+  const handleNameInput = e => {
+    setUserName(e.target.value);
+  };
+
+  const handleEmailInput = e => {
+    setUserEmail(e.target.value);
+  };
+
+  const handlePasswordInput = e => {
+    setUserPassword(e.target.value);
+  };
+
+  const handleRegistration = e => {
+    e.preventDefault();
+    performRegistration(userName, userEmail, userPassword)
```

Comment:
It's better to keep consistency, if you use async\await above, why not use it here?

Reply:
my bad, sorry